styles.less

```diff
@@ -323,7 +323,32 @@ body {
 		}
 
 		.slide-7 {
-			
+      // Previously at the end of the LESS file
+      .slide-7-dark-cloud {
+        position: absolute;
+        width: 15%;
+        height: 15%;
+        left: 74vw;
+        bottom: 78vh;
+        transition: 300ms 500ms;
+        /* Waiting to apply until cloud background is removed
+        -webkit-box-shadow: 9px 9px 7px 0px rgba(50, 50, 50, 0.75);
+        -moz-box-shadow:    9px 9px 7px 0px rgba(50, 50, 50, 0.75);
+        box-shadow:         9px 9px 7px 0px rgba(50, 50, 50, 0.75);
+       */
+
+
+      }
+
+      .slide-7-light-cloud {
+        position: absolute;
+        width: 22%;
+        height: 22%;
```

Comment:
Percentages should be fine here since these are sitting in the background slide, which has a width of 100vw. Just a heads up though - you may have to use units like 22vw and 22vh on other areas of the code because the containers will sometimes be wider than the screen.